lib/db.js

```diff
@@ -474,12 +474,9 @@ module.exports = (
     }
     return P.all(promises)
       .spread((devices, redisTokens) => {
+        log.info({ op: 'db.devices.redisSessionTokens', hasRedisTokens: !! redisTokens })
```

Comment:
Is it ever possible for `redisTokens` to be an empty object?

Reply:
Yep.

Reply:
I'll close this for now, I'm in the middle of something I don't want to break off from at the moment.